suite-native/notifications/src/components/TransactionNotificationDescription.tsx

```diff
@@ -17,24 +15,16 @@ const addressContainerStyle = prepareNativeStyle(_ => ({
 }));
 
 export const TransactionNotificationDescription = ({
-    amount,
+    formattedAmount,
     prefix,
     targetAddress,
-    networkSymbol,
 }: TransactionNotificationDescriptionProps) => {
     const { applyStyle } = useNativeStyles();
 
     return (
         <Box flexDirection="row">
-            <CryptoAmountFormatter
-                value={amount}
-                network={networkSymbol}
-                isBalance={false}
-                color="textSubdued"
-                variant="label"
-            />
             <Text color="textSubdued" variant="label">
-                {` ${prefix} `}
+                {`${formattedAmount} ${prefix} `}
```

Comment:
```suggestion
                {formattedAmount} {prefix}
```

Reply:
Same as the last time, the trailing whitespace is needed for correct formatting.

Reply:
53e695678